### DISCUSSION TITLE
Slack response from Support teams

### DISCUSSION THREAD

**Discussion by kieran-bellew** [serverless/serverless#12849] [Q&A]
We've made multiple inquiries about trying to increase our credits on AWS marketplace. We were initially directed to AWS support from the Serverless support team via chat, but then sent back as it's on them to cancel the contract. We inquired via chat and email and were told it was being worked on but we have yet to hear back after a week or two. The Serverless dashboard is useless as a result. You would expect to see the results from credits paid for at least, but that's not the point of this discussion. We are trying to purchase 50+ credits. It seems like we should avoid any dealings through AWS Marketplace since it's a bit rigid, but this is quite disappointing, regardless. Has anyone had any luck contacting support?

**Comment by garethmcc** [comment]
@kieran-bellew Since we have been discussing by email, I am going to go ahead and close this discussion.
